packages/libs/components/src/map/BoundsDriftMarker.tsx

```diff
@@ -30,12 +39,26 @@ export default function BoundsDriftMarker({
   popupContent,
   popupClass,
 }: BoundsDriftMarkerProps) {
-  const [displayBounds, setDisplayBounds] = useState<boolean>(false);
   const map = useMap();
   const boundingBox = new LatLngBounds([
     [bounds.southWest.lat, bounds.southWest.lng],
     [bounds.northEast.lat, bounds.northEast.lng],
   ]);
+
+  const boundsRectangle = L.rectangle(boundingBox, {
+    color: 'gray',
+    weight: 1,
+  });
+  useEffect(() => {
+    /**
+     * Prevents an edge case where the boundsRectangle persists if simultaneously a marker is hovered
+     * and a user changes the viewport
+     */
+    return () => {
```

Comment:
While I normally like the `() => { }` syntax, it's nice for newbies if this is

`return function cleanup() { ... }`